src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -712,9 +775,19 @@ public void onIndexModule(IndexModule indexModule) {
                     ciol,
                     evaluator,
                     dlsFlsValve::getCurrentConfig,
-                    dlsFlsBaseContext
+                    dlsFlsBaseContext,
+                    resourceAccessHandler
                 )
             );
+
+            // Listening on POST and DELETE operations in resource indices
+            ResourceSharingIndexListener resourceSharingIndexListener = ResourceSharingIndexListener.getInstance();
+            resourceSharingIndexListener.initialize(threadPool, localClient, auditLog);
+            if (RESOURCE_INDICES.contains(indexModule.getIndex().getName())) {
+                indexModule.addIndexOperationListener(resourceSharingIndexListener);
+                log.warn("Security plugin started listening to operations on resource-index {}", indexModule.getIndex().getName());
```

Comment:
Why is this a warning? Warning log messages have the potential to raise concerns amongst admins which monitor the logs.

Reply:
i followed pattern from other reference in the same class but that no longer seem to exist. I will change it to info.